<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>0</class>
 <widget class="QDialog" name="0">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>236</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Range (V+/-)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="range">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20MV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50MV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100MV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>200MV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>500MV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1V</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2V</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5V</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10V</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20V</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Time base (ns)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Samples</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="channel"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Channel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>M1 Velocity</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>M2 Velocity</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="vel1">
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="vel2">
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="okBtn">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="samps">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="timebase">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>1000000000</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
